i18n/i18n_app.properties

```diff
@@ -682,6 +682,4 @@ attribute_to_search=attribute to search
 attributes_to_search=attributes to search
 Yes=Yes
 No=No
-save failed=Save failed
-last_date_in_isolation=Last date in isolation
-last_date_in_quarantine=Last date in quarantine
\ No newline at end of file
+save failed=Save failed
```

Comment:
mangler det underscore her?

Reply:
Fixed - this was unrelated to the current code change.